### PR TITLE
fix(lib/doc): fix jsdoc and typo

### PR DESCRIPTION
**Pull request by dev-itsheng**
When I read Koa's source, I noticed some typo errors in JSDoc and documentation, so I tried to fix them.

### CODE REVIEW

**Review by 3imed-jaberi — APPROVED** [review]

**Review by etroynov — APPROVED** [review]

File: docs/koa-vs-express.md [hunk 1]
```diff
@@ -39,7 +39,7 @@
 ## Does Koa replace Connect?
 
   No, just a different take on similar functionality
-  now that generators allow us to write code with less
```
Comment (3imed-jaberi) [inline docs/koa-vs-express.md:42]: It would be better to keep `less` over `fewer`!

Comment (JonFranchi) [reply docs/koa-vs-express.md:42]: fewer refers to countable, which the number of callbacks would be.

**Comment by etroynov** [comment]
@hustLer2k @gxkl 
Hi,

Can you pls check this changes? Thx

File: docs/koa-vs-express.md (hunk 1 above)
Comment (hustLer2k) [reply docs/koa-vs-express.md:42]: this change is fine by me

**Review by hustLer2k — APPROVED** [review]

**Review by hustLer2k — APPROVED** [review]
# +1

**Review by fengmk2 — APPROVED** [review]

**Pull request merged**